### DISCUSSION TITLE
Push Notification for IOS/Android

### DISCUSSION THREAD

**Discussion by bonsaibauer** [lexiconhq/lexicon#48] [Lexicon Version 1] [answered]
I have a question. I'm currently planning a project using Discourse for a volunteer movement focused on environmental protection. In order for it to become user-friendly and accessible to a larger audience, we would need a native app with push notifications. Is there an estimated timeline or current progress on when this might be implemented, or any way to expedite the process?

Thank you very much @Kinostrome @starjustice @jovitazhang @gabriellaputri @SimonBiggs @shafique17z

**Comment by jovitazhang** [comment]
Hi @bonsaibauer we're currently planning for the BETA launch within next month including the push notifications. 
Keep an eye on our social media for updates about the BETA launch: 	
- ProductHunt Page: https://kfox.io/lexph
- Marketing Site: https://lexicon.is/
- Instagram: https://www.instagram.com/getlexicon/
- Twitter: https://twitter.com/getlexicon

Thank you!

**Comment by jovitazhang** [comment, marked as answer]
Hi @bonsaibauer hope you doing great!

We're thrilled to inform you that we already launched the V2 beta, where we had push notifications support now. 
you could give it a whirl, we'd love to hear your thoughts! Your feedback helps us improve and ensures we're delivering what you need and want.

You can get started by checking out our v2 beta documentation on the Discourse plugin and push notifications here: https://docs.lexicon.is/version-2.0.0-beta/discourse-plugin/.

If you have any questions or need assistance, our dedicated support team is here to help. Feel free to reach out to us!
